examples/with-react-i18next/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "with-react-i18next",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "nodemon",
+    "build": "next build && tsc --project tsconfig.server.json",
+    "start": "cross-env NODE_ENV=production node dist/index.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "i18next": "^17.0.4",
+    "i18next-express-middleware": "^1.8.0",
+    "next": "^8.1.0",
```

Comment:
I think it would be great if example is in the newest version (9). This would also remove the need for ts configuration and types